app/src/main/java/io/qtalk/qgamejsinterfacetester/core/InteractionType.kt

```diff
@@ -2,5 +2,6 @@ package io.qtalk.qgamejsinterfacetester.core
 
 enum class InteractionType {
     WEB_SHARING,
-    IN_CALL
+    IN_CALL,
+    WEBRTC
```

Comment:
rename to: 
```suggestion
    WEB_RTC
```

Reply:
@shahsurajk I think I had a discussion on this with @mishraanuruddh  and We went with WEBRTC as a single word. We've kept it that way in the QTalk client too.

Reply:
https://quiph.slack.com/archives/C743GE5B5/p1601459946000600 `WONTFIX`